### ISSUE TITLE
How to animate a view model value in the view?

### ISSUE THREAD

**Issue by Zenorbi** [aurelia/framework#395]
Given the following template:
```
<span>${foo | numberFormat: "0.0"}</span>
```
I would like to animate `foo`. How should I start working on an animator?

I tried value converters, but a value converter does not have the ability to change the piped value without it being called from aurelia.

Then I looked at how `throttle`, `debounce` work and tried binding behaviors like so:
```
<span>${foo & animate:200}</span>
```
Worked like charm, the only problem is, that I cannot write this:
```
<span>${foo & animate:200 | numberFormat: "0.0"}</span>
```
I then hacked together this really awkward code, and it works, but I feel like there is a better solution, I just don't know it.
```
<span random-attr.bind="foo & animate:200:'animatedFoo'">${animatedFoo | numberFormat: "0.0"}</span>
```
Basically, the `animate` binding behavior writes into `source.bindingContext[animatedPropertyName]`, where the `animatedPropertyName` is the `'animatedFoo'`.

I really think I missed something. Can you please give me a direction, maybe I just missed something from binding behaviors?

**Comment by Zenorbi** [comment]
I did sort of answer my question at http://stackoverflow.com/questions/36654673/create-a-new-binding-context-for-a-custom-attribute/36692609#36692609

Feel free to close this issue if the answer is good, I would however document this feature, maybe in the cheat sheet.

**Comment by Vheissu** [comment]
@EisenbergEffect Looks like this one can be closed.
